### ISSUE TITLE
error caused by specifying bee-api-url with trailing slash

### ISSUE THREAD

**Issue by significance** [ethersphere/swarm-cli#37]
the error does *not* occur with the trailing slash removed, i.e. `-bee-api-url http://localhost:1633` 

```
sig :: Code/swarm2/frontpage.github.io 1 » FRONT=true && swarm-cli --identity="fp-wallet" --password="..." feed upload --path "./dist" --bee-api-url http://localhost:1633/ --topic "0000000000000000000000000000000000000000000000000000000000000000"
```

```
r [Error]: write EPIPE
    at t.safeAxios (/Users/sig/.nvm/versions/node/v14.6.0/lib/node_modules/@ethersphere/swarm-cli/dist/index.js:3:251087)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.t.upload (/Users/sig/.nvm/versions/node/v14.6.0/lib/node_modules/@ethersphere/swarm-cli/dist/index.js:3:236117)
    at async g.uploadDirectory (/Users/sig/.nvm/versions/node/v14.6.0/lib/node_modules/@ethersphere/swarm-cli/dist/index.js:3:1122225)
    at async g.run (/Users/sig/.nvm/versions/node/v14.6.0/lib/node_modules/@ethersphere/swarm-cli/dist/index.js:3:1121413)
    at async u.runUpload (/Users/sig/.nvm/versions/node/v14.6.0/lib/node_modules/@ethersphere/swarm-cli/dist/index.js:3:1099480)
    at async u.run (/Users/sig/.nvm/versions/node/v14.6.0/lib/node_modules/@ethersphere/swarm-cli/dist/index.js:3:1099091)
    at async Object.handler (/Users/sig/.nvm/versions/node/v14.6.0/lib/node_modules/@ethersphere/swarm-cli/dist/index.js:3:659211)
```

**Comment by agazso** [comment]
This is already fixed in bee-js here: https://github.com/ethersphere/bee-js/pull/203

When we use that version this issue will be fixed automatically.

**Comment by agazso** [comment]
Fixed by #46
